src/astt_gui/templates/index.html

```diff
@@ -226,26 +267,31 @@ <h4>Functional State</h4>
                         <div class="container">
                           <div class="row">
                             <div class="col">
-                              <button onclick="openInitModal('')" class="btn btn-primary"><i class='bx bxs-right-arrow'></i> Initialize System</button>
+                              <button onclick="openInitModal('')" class="btn btn-primary" title="This button initializes the system"> <i class='bx bxs-right-arrow'></i> Initialize System</button>
```

Comment:
This looks good and I tested it locally however, we can extend the statements to give more information about what the button does.

eg . Clicking this button will request for the machine's password.
	if the password is correct: 
	Then the antenna/simulator will be started and subscriptions to sensors is made on the antenna.
	then the initial values of sensors will be populated above on the GUI

Reply:
Will do.